aws/setup.go

```diff
@@ -0,0 +1,307 @@
+package aws
+
+import (
+	"context"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
+	"github.com/giantswarm/e2etemplates/pkg/e2etemplates"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/e2esetup/aws/env"
+)
+
+const (
+	awsOperatorArnKey     = "aws.awsoperator.arn"
+	awsResourceValuesFile = "/tmp/aws-operator-values.yaml"
+	credentialName        = "credential-default"
+	credentialNamespace   = "giantswarm"
+	provider              = "aws"
+)
+
```

Comment:
Only the Setup method changes. The rest of the code is copied from aws-operator. Context is passed in from the operator.

Reply:
Cool I wanted to do it in the first place, but I didn't want to introduce too many changes at once 👍